scrapy/addons.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, crawler: "Crawler") -> None:
         self.crawler: "Crawler" = crawler
         self.addons: List[Any] = []
 
-    def load_settings(self, settings: Settings) -> None:
-        """Load add-ons and configurations from a settings object.
+    def apply_settings(self, settings: Settings) -> None:
```

Comment:
heh, should we name this method update_settings, based on what it actually does?

Reply:
Well, the old and the current name reflect that it reads the `ADDONS` setting, not that it runs add-ons' `update_settings` methods, so I'm not sure.

Reply:
Isn't updating the settings the single useful piece of work this method does?

Reply:
Technically, this has nothing to do with this PR, so if there is no easy agreement (cc @Gallaecio), let's revert the name change and handle it separately :)

Reply:
I don’t care, to me this method could be private and then the name does not matter that much :)